### ISSUE TITLE
pretrained models and baseline network

### ISSUE THREAD

**Issue by yifengji** [llcshappy/Monocular-3D-Human-Pose#3]
can you upload these module? I can't download these module,or can you send to my email .my email address is yifengji1020@163.com. Thanks.

**Comment by llcshappy** [comment]
We will release the pre-trained models. We are working on organizing them.

**Comment by ctycty361** [comment]
Hi, I clicked the link in the webpage of the pre-trained models (3D Label Generator), but it turns out that there is no webpage. Can you send it to my email, my email address is ctycty361@outlook.com.

**Comment by AcKnight** [comment]
Hi, thanks for your brilliant work! I have the same problem with @ctycty361 , so could you also send the pretrained model to my email(xiangr@buaa.edu.cn)? thx again

**Comment by amrbenattia** [comment]
hello, thanks for great work, however I would like to get the pretrained model,  could you sent to ama49@fayoum.edu.eg

Thanks in advance
